.travis.yml

```diff
@@ -3,10 +3,12 @@ python:
     - "3.4"
 cache: "pip"
 install:
-    - "pip install ."
+    - "pip install .[tests]"
 script:
     - "flake8 ."
-    - "py.test pythoncz_tests"
+    - "pytest --cov-report="
```

Comment:
I see no reasonable output in the Travis CI build logs. Is it desired? I know that coveralls will update the PRs etc., but I would like to be able to see the information when i looks at the Travis CI log.